docs/source/configuration/accounts_table

```diff
@@ -100,6 +100,26 @@
     :default: draft,
 
 
+.. _replied-tags:
+
+.. describe:: replied_tags
+
+     list of tags to automatically add to replied messages
+
+    :type: string list
+    :default: replied,
+
+
+.. _passed-tags:
+
+.. describe:: passed_tags
+
+     list of tags to automatically add to passed messages
+
+    :type: string list
+    :default: passed,
```

Comment:
stray comma,

Reply:
I followed the existing style for sent and draft here, assuming the comma is there to enforce (or illustrate) the list type.